catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -371,11 +367,10 @@ public DeleteResponse version(DeleteResponse deleteResponse)
       }
     }
 
-    CreateResponse createResponse =
```

Comment:
If possible I would prefer to have this kept in.

Reply:
Should I suppress the sonarqube finding?